### ISSUE TITLE
RC6 and RC5 does not start on my windows 10

### ISSUE THREAD

**Issue by metalgigio** [LMMS/lmms#4405]
Hi,
as subject RC6 and RC5 does not start on my windows 10. It appears splash screen but the program does not load.
1.1.3 version works correctly.
Theres some log i can check to se what happen?

**Comment by lukas-w** [comment]
Unfortunately, our debugging options on Windows are very limited at this time. What happens after the splash screen shows? Does LMMS crash, do you get any error messages?

**Comment by metalgigio** [comment]
On RC6  splash screen appears, then show some loading message on bottom left and then disappear but program does not start. No error messages.
RC1, RC2, RC3, RC4 start correctly, RC5 does not even show splash screen.
I use Windows 10 64bit updated to last version

**Comment by PhysSong** [comment]
I guess you're using PotrAudio. I've done a quick test:
1. Open LMMS 1.1.3 and set audio.backend to PortAudio
2. Open RC5 or later, it will crash
I'm not sure it's the identical issue, but I guess so because PortAudio have been disabled since RC5.

**Comment by metalgigio** [comment]
this is my lmmsrc.xml:

```
<?xml version="1.0"?>
<!DOCTYPE lmms-config-file>
<lmms version="1.2.0-rc4">
  <MidiAlsaRaw device=""/>
  <app nommpz="0" configured="1" displaydbfs="0" nomsgaftersetup="0"/>
  <audioportaudio backend="Windows WASAPI" device="Altoparlanti (Dispositivo High Definition Audio)"/>
  <audiosdl device=""/>
  <mixer framesperaudiobuffer="256" hqaudio="0" mididev="WinMM MIDI" audiodev="PortAudio"/>
  <paths stkdir="data:/stk/rawwaves/" fldir="C:/Users/gigi\" sf2dir="C:/Users/gigi/lmms/samples/sf2/" gigdir="C:/Users/gigi/lmms/samples/gig/" laddir="C:/Program Files/LMMS\plugins\ladspa\" defaultsf2="" artwork="data:/themes/default/" vstdir="C:\Users\gigi\Documents\VST\" backgroundartwork="" workingdir="C:/Users/gigi\lmms\"/>
  <tooltips disabled="0"/>
  <ui oneinstrumenttrackwindow="0" syncvstplugins="0" enableautosave="0" printnotelabels="0" manualchannelpiano="0" smoothscroll="0" compacttrackbuttons="0" disablechannelactivityindicators="0" displaywaveform="0" animateafp="0"/>
  <recentfiles>
    <file path="C:/Users/gigi/OneDrive/lmms/test2.mmpz"/>
    <file path="C:/Users/gigi/OneDrive/lmms/test1.mmpz"/>
    <file path="data:/projects/templates/default.mpt"/>
    <file path="C:/Users/gigi/Desktop/prova1.mmpz"/>
    <file path="C:/Program Files/LMMS/data/projects/Covers\CapDan-Infinity2010.mmpz"/>
    <file path="C:/Users/gigi/lmms/recover.mmp"/>
  </recentfiles>
</lmms>

```

**Comment by metalgigio** [comment]
If the problem is related to PortAudio what should i do?
Do i have to disable it using 1.1.3?

**Comment by PhysSong** [comment]
> Do i have to disable it using 1.1.3?

If you use RC5 or RC6 now, please do so. FYI, we're going to work around the issue in RC7.

**Comment by metalgigio** [comment]
I confirm that now RC6 start correctly by setting audio to SDL in 1.1.3.
Thanks a lot

**Comment by PhysSong** [comment]
Fixed via #4407.
